pb_node_images_management-vm.yml

```diff
@@ -31,6 +31,8 @@
     - vars/fawkes.yml
   roles:
     - packages
+    - sqlite3
```

Comment:
Moving sqlite from a role to a package makes it so that if we add gitea to other systems then we have to duplicate defining the packages that are installed. If we keep things like this as a role, and include it as a dependency of the gitea role, then anything that installs gitea will also install sqlite.

Reply:
Makes sense.

I assume "other systems" mean other artifacts we make?